.github/workflows/prebuild.yml

```diff
@@ -1,5 +1,7 @@
 name: Prebuild
 on:
+  pull_request:
+    branches: [main]
```

Comment:
```suggestion
```

Prebuilds should only be triggered on-demand.

Reply:
Yep, it was a temp hack to test the workflow before merging

Reply:
Ah, you can just trigger it from a specific branch.

Reply:
<img width="350" alt="image" src="https://github.com/user-attachments/assets/7f1f2deb-5637-4ad2-b9a3-01b566a2425f">

Reply:
Running one here: https://github.com/holepunchto/udx-native/actions/runs/11274892535